### ISSUE TITLE
Re-add the "accessibility instructions" modal

### ISSUE THREAD

**Issue by Kataract** [ckeditor/ckeditor5#1014]
## Is this a bug report or feature request? (choose one)

🆕 Feature request

## 💻 Version of CKEditor

CKEditor 5 builds

## 📋 Steps to reproduce

1.  Press alt+0 while in the text area (which used to bring up the accessibility instructions modal in CKE4)

## ✅ Expected result

Accessibility instructions modal pops up

## ❎ Actual result

I get a degrees symbol on my Mac inserted in the text area 😅

---

If you'd like to see this feature implemented, add a 👍 reaction to this post.

**Comment by Reinmar** [comment]
Thanks for the report. Indeed, CKEditor 4 has such a modal:

![image](https://user-images.githubusercontent.com/156149/39989101-59c13910-5769-11e8-8dd5-da3915e0eabc.png)

We may need a similar one for CKEditor 5.

I wonder, though, how many apps handle <kbd>Alt</kbd>+<kbd>0</kbd>? I checked a couple of places (GH, Slack, Google Docs) and none of them showed anything. Does that mean that they don't implement some important accessibility feature? Or is such a modal something specific for CKEditor 4?

cc @Comandeer @oleq @dkonopka

**Comment by oleq** [comment]
Both Gmail and GitHub implement this feature, but it's under <kbd>⇧ Shift</kbd>+<kbd>/</kbd> (or simply <kbd>?</kbd>) which only makes sense when there's no focus in the text input field, which, again, does not make any sense in a text editor ;-)

That's most likely why we used <kbd>Alt</kbd>+<kbd>0</kbd> in v4.

![image](https://user-images.githubusercontent.com/1099479/40046208-af1fa7de-582c-11e8-80c1-d2951194ded2.png)

![image](https://user-images.githubusercontent.com/1099479/40046228-bab54b44-582c-11e8-81ee-a9a6d906e087.png)

**Comment by mlewand** [comment]
There are many variations, but most commonly apps use `/` key today (as opposed to F1 back in a day 🙂).

As @oleq mentioned there are apps using `shift+/`, I know it's also used in Chrome devtools. `alt+/` is also true for GitHub, Bitbucket. `ctrl+/` or `cmd+/` is used in Slack, Gdocs.

IMO the best option for us is `ctrl+/` or `cmd+/`. Actually this is also happens to be a hotkey that [we were planning to use in CKE4 keystrokes dialog](https://github.com/ckeditor/ckeditor-dev/pull/569/files#diff-1c84beafdc76243413cb639e74498bc5R53).

**Comment by s-at-dq** [comment]
Regarding the shortcut for the modal help dialog:

- The screen reader Orca uses the 0 key for opnening context menus (right click menus).
- Depending on the keyboard layout, the key for slash ("/") is only available using the shift key or another modifier.
  - On the normal german layout, you need to press Shift + 7 to get it.
  - On neo2, you need to press Mod3 + I.

**Comment by raq007** [comment]
+1 to the feature request.
